app/Spago.hs

```diff
@@ -31,11 +30,20 @@ main = withUtf8 $ do
   Env.setEnv "GIT_TERMINAL_PROMPT" "0"
 
   (command, globalOptions@GlobalOptions{..}) 
-    <- CLI.optionsExt "" "" "Spago - manage your PureScript projects" (Text.pack spagoVersion) CLI.parser
+    <- CLI.options "Spago - manage your PureScript projects" CLI.parser
+
+  let handleDefault :: MonadIO m => ShowVersion -> m ()
+      handleDefault = \case
+        DoShowVersion -> CLI.echo spagoVersion *> exitSuccess
+        NoShowVersion -> pure ()
```

Comment:
I think we should show the help message and fail here?

Reply:
Please see the explanation below.